ui/shared/components/panel/acmg/AcmgCriteria.jsx

```diff
@@ -225,6 +242,13 @@ class AcmgCriteria extends React.PureComponent {
           )}
         <Button primary onClick={this.submitForm}>Submit</Button>
         <Button onClick={this.clearFields} color="grey">Clear Form</Button>
+        <Confirm
+          key="confirm"
+          content={confirmDialog}
+          open={!!confirmDialog}
+          onConfirm={this.submitClearedForm}
+          onCancel={this.hideConfirmDialog}
```

Comment:
You may want to specify the attributes for `cancelButton` (e.g. `Don't Clear`) and `confirmButton` (e.g. "Clear Form" to remove any ambiguity.

Reply:
No, the defaults are just fine and its better to have a simpler, more consistent application than an overly complex and hard to maintain one